src/custom/components/NotificationBanner/index.tsx

```diff
@@ -1,15 +1,17 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 import { Colors } from 'theme/styled'
 import { X } from 'react-feather'
 import { MEDIA_WIDTHS } from '@src/theme'
+import { useLocalStorage } from 'hooks/useLocalStorage'
 
 type Level = 'info' | 'warning' | 'error'
 
 export interface BannerProps {
   children: React.ReactNode
   level: Level
   isVisible: boolean
+  changeOnProp?: any
```

Comment:
Avoid `any` when possible

```suggestion
  changeOnProp?: string
```